cosmos/errors.go

```diff
@@ -0,0 +1,14 @@
+package cosmos
+
+import "github.com/cosmos/cosmos-sdk/types"
+
+const (
+	CodeForbidden types.CodeType = 403
+	CodeNotFound                 = 404
+	CodeExist                    = 409
```

Comment:
nice. but why 4xx? is there any reason?

Reply:
is just an example. I took it from http rest api , but we can came up with our own codes

Reply:
good idea to reuse the http ones ;)